app/src/pages/project/TracesTable.tsx

```diff
@@ -229,8 +253,33 @@ export function TracesTable(props: TracesTableProps) {
         ...rootSpan.descendants.edges.map(({ node }) => node),
       ]);
       // Unwrap the root span from the span tree and return it
-      const [root] = spanTreeToNestedSpanTableRows(spanTree);
-      return root;
+      const [root] = spanTreeToNestedSpanTableRows({
+        children: spanTree,
+      });
+      type SpanRowType = typeof root & IAdditionalSpansRow;
+      // check if there are more spans in the tree than is loaded
+      const numSpansNotLoaded = rootSpan.trace.numSpans - NUM_DESCENDANTS - 1;
+      if (numSpansNotLoaded > 0) {
+        root.children = [
+          ...root.children,
+          // We add a dummy span here to indicate that there are more spans in the tree
+          {
+            ...root,
+            // Indicate that this is an additional row, not a span
+            __additionalRow: true,
+            name: `+ ${numSpansNotLoaded} more span${numSpansNotLoaded > 1 ? "s" : ""}`,
+            id: `additional-${root.id}`,
+            // Clear out the span info
+            input: { value: "" },
+            output: { value: "" },
+            metadata: null,
+            spanAnnotations: [],
+            documentRetrievalMetrics: [],
+            children: [],
+          } as SpanRowType,
```

Comment:
satisfies SpanRowType would be safer. Would ensure you get all the keys / val types

Reply:
Unfortunately not as it's not letting me expand the type that way

```
Object literal may only specify known properties, and '__additionalRow' does not exist in type 'NestedSpanTableRow<{ readonly cumulativeTokenCountCompletion: number | null; readonly cumulativeTokenCountPrompt: number | null; readonly cumulativeTokenCountTotal: number | null; readonly documentRetrievalMetrics: readonly { ...; }[]; ... 11 more ...; readonly trace: { ...; }; }>'.ts(2353)
```